packages/actionmenu/src/css/index.js

```diff
@@ -32,6 +32,8 @@ export default {
     padding: `${vars.style.menuPaddingVert} 0`,
     minWidth: '160px',
     maxWidth: '320px',
+    maxHeight: '277px',
+    overflowY: 'auto',
```

Comment:
For some reason, this or something else in this branch is affecting more than just Y overflow.  The X scrollbar is now always there, whether needed or not.  

For example:
<img width="325" alt="Screen Shot 2019-10-01 at 11 20 01 AM" src="https://user-images.githubusercontent.com/552044/65985397-f02b8500-e43e-11e9-9686-47d4f935fe13.png">

Reply:
constraining the height on the ActionMenu is causing lots of issues because of the nested menus. i'll remove that from the ActionMenu